### PR TITLE
Added in submenus

### PR DESCRIPTION
**Pull request by mikestowe**
Updated for new pages launch on Project Calico website

**Let's review so that merge takes place at same time as new pages launch ;)**

### CODE REVIEW

**Comment by mikestowe** [comment]
@robbrockbank @ozdanborne ready for review, and merge (7/20/2017 or later).

This PR:
- Updates site navigation to match ProjectCalico.org site
- Adds in breadcrumbs
- Adds in GitHub Edit Link - see #781 
- Adds in documentation rating mechanism
- Provides link to feedback form on ProjectCalico.org
- Get More Help Links (Slack/ MVP)

File: _layouts/docwithnav.html [hunk 1]
```diff
@@ -168,9 +191,30 @@
...
+              <div style="text-align: left; padding: 18px 0 9px 0; min-width: 500px; overflow-x: hidden;">
+              <!-- START BREADCRUMBS -->
+				{% assign crumbs = page.url | remove:'/index.html' | split: '/' | shift | pop %}
+				{% assign crumbs_version = page.url | split: '/' | first %}
+				{% for crumb in crumbs offset: 1 %}
+				  {% if forloop.last %}
+					{% comment %}/ {{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}{% endcomment %}
+					<a href="/{{crumbs_version}}{% assign crumb_limit = forloop.index | plus: 1 %}{% for crumb in crumbs limit: crumb_limit %}{{ crumb | append: '/' }}{% endfor %}">{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a>
+				  {% else %}
+					<a href="/{{crumbs_version}}{% assign crumb_limit = forloop.index | plus: 1 %}{% for crumb in crumbs limit: crumb_limit %}{{ crumb | append: '/' }}{% endfor %}">{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a> <span style="color: #999;">/</span>
```
Comment (ozdanborne) [inline _layouts/docwithnav.html]: needs `{{site.basurl}}`

File: _layouts/docwithnav.html [hunk 2]
```diff
@@ -82,10 +83,32 @@
           <div id="tigera-navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
               <li><a href="https://projectcalico.org">Home</a></li>
-              <li class="active"><a href="#"><span>Docs</span></a></li>
-              <li><a href="https://projectcalico.org/community">Community</a></li>
-              <li><a href="https://projectcalico.org/blog">Blog</a></li>
-              <li><a href="https://projectcalico.org/support">Support</a></li>
+              <li class="active"><a href="https://projectcalico.org/resources"><span>Resources</span></a>
+                <ul>
+                    <li><a href="http://docs.projectcalico.org/latest/getting-started/"><i class="fa fa-tachometer" aria-hidden="true"></i> &nbsp; Getting Started</a></li>
```
Comment (ozdanborne) [inline _layouts/docwithnav.html]: needs `{{site.baseurl}}`

File: _layouts/docwithnav.html [hunk 3]
```diff
@@ -82,10 +83,32 @@
...
+                    <li><a href=""><i class="fa fa-paper-plane-o" aria-hidden="true"></i> &nbsp; Online Workshops</a></li>-->
+                    <li><a href="https://www.projectcalico.org/videos"><i class="fa fa-video-camera" aria-hidden="true"></i> &nbsp; Videos &amp; Webinars</a></li>
+                    <li><a href="{{site.baseurl}}/latest/releases/"><i class="fa fa-clock-o" aria-hidden="true"></i> &nbsp; Releases</a></li>
+                    <!-- <li><a href="/mvp"><i class="fa fa-star" aria-hidden="true"></i> &nbsp; MVP Program</a></li> -->
+                    <li><a href="https://www.projectcalico.org/blog"><i class="fa fa-bookmark" aria-hidden="true"></i> &nbsp; Blog</a></li>
+                </ul>
+              </li>
+              <li><a href="https://www.projectcalico.org/community">Community</a>
+                <ul>
+                    <li><a href="https://www.projectcalico.org/community#slack"><i class="fa fa-slack" aria-hidden="true"></i> &nbsp; Slack</a></li>
```
Comment (ozdanborne) [inline _layouts/docwithnav.html]: Could be more clear if we linked them directly to http://slack.projectcalico.org/ ?

File: _layouts/docwithnav.html [hunk 4]
```diff
@@ -168,9 +191,40 @@
...
+
+					  {% if breadcrumb_exists != '' and breadcrumb_exists != nil and breadcrumb_exists.size > 0 %}
+						<li class="breadcrumb-item"><a href="{{site.baseurl}}/{{crumb_url}}">{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a></li>
+					  {% endif %}
+					{% endfor %}
+				 {% endif %}
+			  </ol>
+              <!-- END BREADCRUMBS -->
+              
+              <a href="https://github.com/projectcalico/calico/tree/master/{{page.path}}" style="color: #999; text-decoration: none; float: right;" target="_blank"><i class="fa fa-github" aria-hidden="true"></i> Edit</a></div>
```
Comment (ozdanborne) [inline _layouts/docwithnav.html]: How bout a more descriptive action than "edit". Like "Edit this page on github"

Comment (robbrockbank) [inline _layouts/docwithnav.html]: Is the leading space deliberate?  Should it be a&nbsp; as with the other heading links?

**Pull request merged**